src/utils/errors.ts

```diff
@@ -423,6 +423,13 @@ export class OpenAIBadVersion extends OpenAIBaseError {
   }
 }
 
+export class OpenAIBadTemperature extends OpenAIBaseError {
+  constructor(temperature: number, maxTemperature: number, minTemperature: number) {
+    const message = `The temperature must be between ${minTemperature} and ${maxTemperature}. You set ${temperature}.`;
```

Comment:
Reuse your constants here?

Reply:
Good question. In the end, I'm not even using this error anymore. I'm doing the validation in the schema. Maybe I'll just remove it.